test/unit/core/action-creator.ts

```diff
@@ -327,12 +327,35 @@ suite('ActionCreator', ({ expect, spy, stub }) => {
     describe('request action creators', () => {
       describe('updateSearch()', () => {
         it('should create an UPDATE_SEARCH action', () => {
-          const data: any = { a: 'b' };
-          const thunk = stub(utils, 'thunk');
+          const search: Actions.Search = { query: 'harry' };
+          const dispatch = spy();
+
+          actions.updateSearch(search)(dispatch);
+
+          expect(dispatch).to.be.calledWith({ type: Actions.UPDATE_SEARCH, ...<any>search });
+        });
+
+        it('should not create an UPDATE_SEARCH action when query string is empty', () => {
```

Comment:
As previously discussed, we've decided against combining these because these test different expectations.